account-management-api/src/main/java/uk/gov/di/accountmanagement/lambda/UpdateEmailHandler.java

```diff
@@ -107,6 +113,13 @@ public APIGatewayProxyResponseEvent updateEmailRequestHandler(
         try {
             UpdateEmailRequest updateInfoRequest =
                     objectMapper.readValue(input.getBody(), UpdateEmailRequest.class);
+            dynamoEmailCheckResultService
+                    .getEmailCheckStore(updateInfoRequest.getReplacementEmailAddress())
+                    .ifPresent(
+                            result ->
+                                    LOG.info(
+                                            "UpdateEmailHandler: Experian email verification status: {}",
+                                            result.getStatus()));
```

Comment:
Is it possible to add a log if it's not present to say the status is PENDING?

Reply:
From what you're saying the assumption is that this lambda is called only after an email check has been requested and can't be called without. 
I don't like adding something like this to a separate lambda unless you know it's been requested but I'm not as close to this as you are so will go with your suggestion and add it.

Reply:
This lambda is called by Home team to update the users email at any point and it is possible that we don't have a result in the database by the time this lambda is called. It is an expected that sometimes we won't have a result for the check at this point. 

Happy to leave it as is though.

Reply:
as-is is with the additional logging now so if you can give it a once over and then I will merge it.